index.html

```diff
@@ -6,15 +6,17 @@
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link rel="stylesheet" href="css/styles.css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://kit.fontawesome.com/a3af588d0f.js" crossorigin="anonymous"></script>
 </head>
 <body>
     <nav>
         <h3 class="dark-purple">My Logo</h3>
```

Comment:
**[OPTIONAL]**
- [ ] It's a better practice that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring that. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of `UX` convention make the navigation in the website way more pleasant and intuitive.

Reply:
That's a great Suggestion, I will implement it right away :D